### PR TITLE
Fluffy the Pink Diyaab - Quartermaster Pet

### PR DESCRIPTION
**Pull request by DarkWolfInsanity**
Exotic pet for the master of shipping and mining. Spawns in their office like any other pet.

fluffy

fluffymove

fluffysleep

### CODE REVIEW

**Comment by SpadesNeil** [comment]
Gotta discard the old map copy and add him back in manually.

**Comment by DarkWolfInsanity** [comment]
It's not an old map copy. It's the exact edit from Dhael's push, just with Fluffy added. Compare them.

**Comment by SpadesNeil** [comment]
According to the conflict, they are not.

**Comment by DarkWolfInsanity** [comment]
It will say that with any little map change because everything shifts.

**Comment by SpadesNeil** [comment]
That doesn't happen if you merge the map correctly.

**Comment by DarkWolfInsanity** [comment]
Well, I'm working on it then.

**Comment by DarkWolfInsanity** [comment]
Unfortunately, that last fix is the best you will probably get using the map merge tool.

**Comment by Cameron653** [comment]
I assume the map problem is fixed, as it only says 253 additions 119 deletions, but you'll need to add

```
#include " code/modules/mob/living/simple_animal/friendly/fluffy.dm"
```
to the .dme file.

Additionally, you should probably change fluffy.dm to fluffy_vr.dm

**Comment by DarkWolfInsanity** [comment]
I was going to leave it as fluffy.dm since no server would probably ever make a simple_animal/friendly/fluffy.dm that we would merge, but alright. I'll fix this up real quick.

File: maps/virgo-1.dmm [hunk 1]
```diff
@@ -631,13 +631,13 @@
...
 "aml" = (/obj/structure/grille,/obj/structure/window/reinforced,/obj/structure/window/reinforced{dir = 4},/obj/structure/window/reinforced{dir = 1},/obj/machinery/door/firedoor/border_only,/turf/simulated/floor/plating,/area/security/detectives_office)
 "amm" = (/obj/machinery/door/airlock/glass_security{id_tag = "detdoor"; name = "Detective"; req_access = list(4)},/obj/machinery/door/firedoor/glass,/obj/structure/cable/green{d1 = 1; d2 = 2; icon_state = "1-2"},/obj/machinery/atmospherics/pipe/simple/hidden/scrubbers,/obj/machinery/atmospherics/pipe/simple/hidden/supply,/turf/simulated/floor/lino,/area/security/detectives_office)
 "amn" = (/obj/structure/grille,/obj/structure/window/reinforced,/obj/structure/window/reinforced{dir = 1},/obj/structure/window/reinforced{dir = 8},/obj/machinery/door/firedoor/border_only,/obj/structure/disposalpipe/segment,/turf/simulated/floor/plating,/area/security/detectives_office)
-"amo" = (/turf/simulated/wall,/area/security/security_lockerroom)
-"amp" = (/turf/simulated/wall,/area/security/security_bathroom)
```
Comment (SpadesNeil) [inline maps/virgo-1.dmm]: This is evidence that you are indeed **not** using the correct map version. Merging this will undo all recent map changes.

Comment (DarkWolfInsanity) [inline maps/virgo-1.dmm]: For the last. Fucking. Time. I took the map. That was used. In Dhael's branch. Which is the branch. You literally JUST MERGED TWO DAYS AGO. IT IS THE EXACT SAME MAP FILE.

**Comment by SpadesNeil** [comment]
Didn't work as planned because Github doesn't like using the online editor for large files.

However https://github.com/VOREStation/VOREStation/pull/814 is how it should look when you add only one item to the map.

Next time, listen to instructions instead of screaming. Issue closed.

**Pull request closed without merging**